### PR TITLE
Users/adi ro h/bug1246 interurban resolution

### PR DESCRIPTION
**Pull request by AdiRoH**
resolving bug #1246 
https://github.com/hasadna/anyway/issues/1246

### CODE REVIEW

**Comment by AdiRoH** [comment]
A file, under an external folder from the code, also includes the bug:
anyway/static/data/news_flash_infographics_mock/mock_data.json includes also בינעירוני , coded to utf-16: \u05d1\u05d9\u05e0\u05e2\u05d9\u05e8\u05d5\u05e0\u05d9 (first line in the file, the value of resolution). 

For resolving the bug, I changed only the code itself, in "anyway" folder. not this file

**Comment by atalyaalon** [comment]
@AdiRoH great work!
Since I added some prs - there is a conflict in anyway/flask_app.py (the code was transferred to another file in [this pr](https://github.com/hasadna/anyway/pull/1455)) - please fix and push changes.

File: anyway/parsers/infographics_data_cache_updater.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def get_cache_info():
 
 def main(update, info):
     if update:
-        logging.info("Refreshing infographics cache...")
+        logging.info("Refreshing infographics cache...")
```
Comment (idosavion) [inline anyway/parsers/infographics_data_cache_updater.py:120]: by mistake?

File: anyway/flask_app.py [hunk 2]
```diff
@@ -1920,6 +1920,7 @@ def oauth_authorize(provider):
         return redirect(url_for("index"))
     oauth = OAuthSignIn.get_provider(provider)
     return oauth.authorize()
+
```
Comment (idosavion) [inline anyway/flask_app.py:1924]: needed?

**Review by idosavion — CHANGES_REQUESTED** [review]
Consider extracting the hebrew value to a constant, will reduce the need to change this value across so many files.

File: anyway/parsers/infographics_data_cache_updater.py (hunk 1 above)
Comment (AdiRoH) [reply anyway/parsers/infographics_data_cache_updater.py:120]: @idosavion is right :) I think that this is a good idea, but there are files, that you can't use this method: 

### anyway/static/data/news_flash_infographics_mock/mock_data.json   
contains words in utf-16
### anyway/tests/accidents_definitional_ynet.tsv:
I'm not sure if there are constant in a tsv file.

**Comment by AdiRoH** [comment]
@idosavion is right :) I think that this is a good idea, but there are files, that you can't use this method: 

### anyway/static/data/news_flash_infographics_mock/mock_data.json   
contains words in utf-16
### anyway/tests/accidents_definitional_ynet.tsv:

**Comment by idosavion** [comment]
of course 😄 , meant only in *.py files.

**Comment by idosavion** [comment]
also, consider amending the commit message into something describing the actual change being made:
`Modify interurban resolution text\extracting interurban resolution text to a constant` or anything similar 😄

**Pull request closed without merging**